packages/log/0.3.1/img/icon.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@mostlyjason @hbharding This is an icon we should review. I think it is one I just picked in the past for playing around.

Reply:
Thanks for catching this. The current icon is using one of our multi-color "application" icons, which used to appear in Kibana's side navigation, but is no longer present. We have an icon for "Logs", but the design team wants to phase out these branded application icons. (2nd row, 7 across. looks like the Sydney Opera House to me 😄 )

![image](https://user-images.githubusercontent.com/847805/89046209-a5a6bd80-d31a-11ea-904e-c1987134c80c.png)

Because this package is "Custom logs", i'm thinking something generic would be appropriate here. Will there be other packages like "Custom metrics" or "Custom foo". I'm leaning towards using the default package icon, which I believe EPM uses when an icon is not specified. So the answer could be don't specify an icon at all!

![image](https://user-images.githubusercontent.com/847805/89047539-8ad54880-d31c-11ea-8ae3-0ca93ac19839.png)

I suspect we will run into this issue again as we develop more custom packages that fill generic purposes, so I like the idea of a reusable solution. Another generic option would be to use the Elastic logo, which would help the package stand out more, and suggest to the user that the package is officially built/maintained by Elastic. This could be useful if we allow community packages in our registry in the future and want to distinguish them visually.

![image](https://user-images.githubusercontent.com/847805/89050214-6c714c00-d320-11ea-9eb7-825134203abc.png)

Reply:
@hbharding As I think this is going to a be package that is pretty often used, I would think it should have an icon that users recognise in comparison to packages without an icon. But lets take this conversation somewhere else. Will ping you.